src/demux/readers/eos/MongoBlock.ts

```diff
@@ -0,0 +1,41 @@
+import { EosAction } from "./interfaces"
+import { Block } from "../../../../index"
+
+export class MongoBlock implements Block {
```

Comment:
Is this class any different than the NodeosBlock? If not, maybe we should just rename the other one to `EosBlock`

Reply:
I was thinking that but there are small differences in the data structure

Reply:
👍